lib/core/src/model.rs

```diff
@@ -89,6 +94,7 @@ impl Config {
             zero_conf_max_amount_sat: None,
             breez_api_key,
             external_input_parsers: None,
+            use_default_external_input_parsers: true,
```

Comment:
Just to check, does the default picknpay parser on testnet also?

Reply:
Parsing the input should work and will result in an `InputType::LnUrlPay`. Paying will not be possible because the LNURL server returns mainnet invoices.